app/controlplane/pkg/data/project.go

```diff
@@ -337,7 +337,11 @@ func (r *ProjectRepo) queryMembership(orgID uuid.UUID, projectID uuid.UUID, memb
 			membership.MemberID(memberID),
 			membership.ResourceTypeEQ(authz.ResourceTypeProject),
 			membership.ResourceID(projectID),
-			membership.ParentIDIsNil(), // Only top-level memberships
+			// include both direct (parent nil) and indirect (parent not nil) memberships
```

Comment:
are you sure this is not going to break anything? Why did we have `membership.ParentIDIsNil(),` before?